.github/workflows/release.yml

```diff
@@ -15,6 +15,9 @@ on:
           - Redeploy
           - Dry Run
 
+env:
+  _AZ_REGISTRY: 'bitwardenprod.azurecr.io'
+
```

Comment:
🎨  `AZ_REGISTRY` was used in previous instances of setting `env:` values. Please consider updating this one to match https://github.com/bitwarden/server/pull/3157/files/efcacac3709d109ab55d2380b0ed000430c8c571#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R14 or vice versa.